src/components/cards-grid-list/index.tsx

```diff
@@ -31,6 +31,7 @@ function CardsGridList({
 	children,
 	isOrdered = false,
 	fixedColumns,
+	gridGap = '24px',
```

Comment:
Now supports an optional `gridGap` prop. This is needed to meet the spec for the downloads CTA, which has a `16px` gap.